### PR TITLE
[risk=low][RW-9836] Rename GKE app creation inside-panel components to "create"

### PR DESCRIPTION
**Pull request by jmthibault79**
This is a followup to https://github.com/all-of-us/workbench/pull/7921

GkeAppConfigurationPanel is a *container* for various types of panel content.  The default content for all GKE Apps is "create" so more appropriate names for the these panel content components are **CreateCromwell** and **CreateRStudio**.

Tested by observing these panels on Local->Test

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [x] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/components/gke-app-configuration-panel.spec.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ import {
   GkeAppConfigurationPanelProps,
   GKEAppPanelContent,
 } from './gke-app-configuration-panel';
-import { DEFAULT_PROPS as RSTUDIO_DEFAULT_PROPS } from './rstudio-configuration-panel.spec';
```
Comment (jmthibault79) [inline ui/src/app/components/gke-app-configuration-panel.spec.tsx:31]: unrelated renaming ... I can revert these if you like

File: ui/src/app/components/gke-app-configuration-panel.tsx [hunk 2]
```diff
@@ -24,24 +24,14 @@ export type GkeAppConfigurationPanelProps = {
...
-} & Omit<CreateGKEAppPanelPropsWithAppType, InjectedProps>;
+} & Omit<CreateGkeAppProps, InjectedProps>;
 
 export enum GKEAppPanelContent {
   CREATE,
   DELETE_UNATTACHED_PD,
   DELETE_GKE_APP,
 }
 
-const CreateGKEAppPanel = ({
```
Comment (jmthibault79) [inline ui/src/app/components/gke-app-configuration-panel.tsx:35]: this is overkill.  It's just a switchCase, so inline that.

File: ui/src/app/components/gke-app-configuration-panel.tsx [hunk 3]
```diff
@@ -136,18 +126,38 @@ export const GKEAppConfigurationPanel = ({
...
-          }}
-        />
-      ),
+      () =>
+        switchCase(
+          appType,
+          [
+            AppType.CROMWELL,
+            () => (
+              <CreateCromwell
```
Comment (jmthibault79) [inline ui/src/app/components/gke-app-configuration-panel.tsx:135]: This naming convention seems more sensible.

panel content = create
type = Cromwell

component -> CreateCromwell

File: ui/src/app/components/gke-app-configuration-panels/create-gke-app-button.spec.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { AppStatus } from 'generated/fetch';
 import { render, screen, waitFor } from '@testing-library/react';
 import { defaultCromwellConfig } from 'app/components/apps-panel/utils';
 import {
-  CreateGKEAppButton,
+  CreateGkeAppButton,
```
Comment (jmthibault79) [inline ui/src/app/components/gke-app-configuration-panels/create-gke-app-button.spec.tsx:8]: more unrelated renaming

File: ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx [hunk 5]
```diff
@@ -26,35 +26,39 @@ import {
...
-import { CreateGKEAppButton } from './create-gke-app-button';
+import { CreateGkeAppButton } from './create-gke-app-button';
 import { DisabledCloudComputeProfile } from './disabled-cloud-compute-profile';
 
 const defaultIntroText =
   'Your analysis environment consists of an application and compute resources. ' +
   'Your cloud environment is unique to this workspace and not shared with other users.';
 
-export interface CreateGKEAppPanelProps {
+export interface CreateGkeAppProps {
```
Comment (jmthibault79) [inline ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx:36]: This is now the full set of props for the generic Create GKE App component

**Review by Peter-Lavigne — APPROVED** [review]
Definitely better, thanks 👍
